examples/a_treeple/treeple_tutorial_CMI.py

```diff
@@ -0,0 +1,149 @@
+"""
+====================================
+Treeple tutorial for calculating CMI
+====================================
+"""
```

Comment:
We should add a short paragraph describing how we actually do the estimate. And maybe what we will show. It will help users acquaint with the example. 

TODO: also link reference to paper when we make preprint online.